source/includes/webhooks/_department_deleted.md

```diff
@@ -0,0 +1,28 @@
+## Department deleted
+
+This web hook fires when departments are deleted from the Greenhouse system from the Configure &raquo; Organization screen. This only happens when the "Remove" button is clicked after clicking the "X" next to an office name. This will not fire individually when an organization is deleted.
+
```

Comment:
I'm not sure the organization sentence is required.  It is, in fact, terrifying.